package.json

```diff
@@ -5,26 +5,31 @@
   "scripts": {
     "serve": "vue-cli-service serve",
     "build": "vue-cli-service build",
+    "test:unit": "vue-cli-service test:unit",
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
     "core-js": "^3.6.5",
     "vue": "^3.0.0",
-    "vue-router": "^4.0.0-0",
-    "vuex": "^4.0.0-0"
+    "vue-router": "^4.0.0-beta.13",
```

Comment:
How forward of you

Reply:
!!

It's just what the CLI did!

Reply:
Even crazier:
```
"vue-jest": "^5.0.0-alpha.4"
```

Reply:
Looks like only beta package is out to support `vue-3.3.0` 
https://github.com/vuejs/vue-test-utils-next/releases

Vue3 has been released only Sept 18, 2020